convert2rhel/initialize.py

```diff
@@ -77,6 +115,13 @@ def run():
     # Initialize logging to stop duplicate messages.
     disable_root_logger()
 
+    # initialize logging
+    initialize_logger()
+
+    # since we now have root, we can add the FileLogging
+    # and also archive previous logs
+    initialize_file_logging("convert2rhel.log", logger_module.LOG_DIR)
+
     from convert2rhel import main
 
     return main.main()
```

Comment:
Given this is called before `main()` what is the safeguard in case we are not running as root?

Reply:
Haven't tested that part... I'm not too involved in the root vs no root thread we had in the past regarding logging and etc... 

Do you think there is a way for us to be on the safe side with this, moving the logging to initialize?

Reply:
Without splitting up main a bit more or something.. not really. We need to create the file handler where we currently do it to ensure that we have root and that we're not running another instance. We can initialize everything else though

Reply:
That is to say, we can revert this to how it was. Just init the logger here
```suggestion

    from convert2rhel import main

    return main.main()
```

Reply:
Ah, alright! Will apply this in a second. Thanks!

Reply:
applied it manually because of unit_tests.